src/gatsby/src/components/advisories/advisoryCard.js

```diff
@@ -248,6 +248,22 @@ const AdvisoryCard = ({ advisory, parkInfoHash }) => {
                   <small><b>{formatDate(advisory.updatedDate)}</b></small>
                 </div>
               )}
+            {/* if "No date" is selected */}
+            {!advisory.isEffectiveDateDisplayed && !advisory.isAdvisoryDateDisplayed && !advisory.isUpdatedDateDisplayed && (
+              formatDate(advisory.updatedDate) ? (
+                <div className="date">
+                  <small>Updated</small>
+                  <small><b>{formatDate(advisory.updatedDate)}</b></small>
+                </div>
+              ) : (
+                formatDate(advisory.advisoryDate) && (
+                  <div className="date">
+                    <small>Posted</small>
+                    <small><b>{formatDate(advisory.advisoryDate)}</b></small>
+                  </div>
+                )
+              )
+            )}
```

Comment:
I always find blocks like these to be hard to read 😓 Chained `&&`s and a multi-line ternary block with more `&&`s. It works the way it is, but it might be a big win for readability to move some of that logic out of the template. 

Maybe have a variable like "`noDateSelected`" and/or a function to handle this logic and return the appropriate block of JSX? 

You don't have to change anything here, but that's something to think about next time, if you find yourself adding a lot of logic in the template 👍

Reply:
My bad, I agree with you 😅 Yeap it has so many conditions ... Updated my PR, hope it got better 🙏